### DISCUSSION TITLE
Please add support for YoLink Relay as a garage door controller.

### DISCUSSION THREAD

**Discussion by steveswales** [dkerr64/homebridge-yolink#104] [Ideas]
I have a YoLink Relay which I've attached to my TOPENS gate controller, and would like to pair it with the YoLink Door Sensor to form a garage door pair.  Can you add Relay to Finger and Garage Door Controller as a supported type?

**Comment by dkerr64** [comment]
Discussion continues in #105, closing this so we only have one thread.
